sourcecode/hub/resources/views/components/mobile-filter-options.blade.php

```diff
@@ -0,0 +1,32 @@
+<div class="offcanvas offcanvas-bottom" id="offcanvasBottomMobile" aria-labelledby="offcanvasBottom" data-bs-backdrop="static" data-bs-keyboard="false">
+    <div class="offcanvas-body">
+        <div class="mb-2">
+            <button
+                id="backButton"
+                class="btn btn-secondary d-lg-none filter-back-button backButton border-0"
```

Comment:
There is no need to hide the back-button on large resolutions, it will display only if the off-canvas content displays